pkg/ccl/backupccl/restore_planning.go

```diff
@@ -1699,6 +1699,9 @@ func doRestorePlan(
 		return err
 	}
 
+	if err := validateRestoredDescriptors(ctx, p.ExecCfg()); err != nil {
+		return errors.Wrap(err, "the restoring cluster contains invalid descriptors, %s")
+	}
```

Comment:
🤔 I wonder if we want an option to ignore these checks.  Should an unrelated descriptor problem stop your attempt to restore?

Reply:
i was going to ask. I think that would be a good idea.

Reply:
Done.